src/Stratis.Bitcoin.IntegrationTests/NodeBuilder.cs

```diff
@@ -739,26 +739,45 @@ public void Rollback(Transaction tx)
             }
         }
 
-#if !NOSOCKET
-
         public void Broadcast(Transaction transaction)
         {
-            using (var node = this.CreateNodeClient())
+            using (var peer = this.CreateNetworkPeerClient())
             {
-                node.VersionHandshake();
-                node.SendMessageAsync(new InvPayload(transaction));
-                node.SendMessageAsync(new TxPayload(transaction));
-                node.PingPong();
+                peer.VersionHandshake();
+                peer.SendMessageAsync(new InvPayload(transaction));
+                peer.SendMessageAsync(new TxPayload(transaction));
+                this.PingPong(peer);
             }
         }
 
-#else
-        public void Broadcast(Transaction transaction)
+        /// <summary>
+        /// Emit a ping and wait the pong.
```

Comment:
I know you're just moving this around, but can you make this explanation a bit clearer?

Reply:
No it's fine, leave it as is for now.

Reply:
Its a quick a dirty method to broadcast a trx, handshake, send inv, send trx, ping to see node is still connected.
Its probably mostly used for testing